session/bootstrap.go

```diff
@@ -225,6 +225,7 @@ const (
 		update_time timestamp NOT NULL,
 		charset text NOT NULL,
 		collation text NOT NULL,
+		UNIQUE INDEX primary_index(original_sql(1024),default_db(1024)) COMMENT "accelerate the speed when add global binding query",
```

Comment:
It may not unique after using the prefix. And it is better to upgrade the table for old tidb servers.